src/utils/event-manager/event-manager.js

```diff
@@ -0,0 +1,97 @@
+import {Manager} from 'hammerjs';
+import WheelInput from './wheel-input';
+import {
+  BASIC_EVENT_ALIASES, EVENT_RECOGNIZER_MAP, RECOGNIZERS, GESTURE_EVENT_ALIASES
+} from './constants';
+
+/**
+ * Single API for subscribing to events about both
+ * basic input events (e.g. 'mousemove', 'touchstart', 'wheel')
+ * and gestural input (e.g. 'click', 'tap', 'panstart').
+ * Delegates event registration and handling to Hammer.js.
+ */
+export default class EventManager {
+  constructor(element, options = {}) {
+    // TODO: support overriding default RECOGNIZERS by passing
+    // recognizers / configs, keyed to event name.
+
+    this._onBasicInput = this._onBasicInput.bind(this);
+    this.manager = new Manager(element, {recognizers: RECOGNIZERS})
```

Comment:
Hunh, since you don't need `mousemove` you can just work completely around the whole `inputClass` thing. Good for you.